thorlcr/activities/keyedjoin/thkeyedjoinslave.cpp

```diff
@@ -1291,6 +1291,7 @@ class CKeyedJoinSlave : public CSlaveActivity, implements IJoinProcessor, implem
                 activity.inactiveStats.sumStatistic(StNumIndexSeeks, keyManager->querySeeks()-startSeeks);
                 activity.inactiveStats.sumStatistic(StNumIndexScans, keyManager->queryScans()-startScans);
                 activity.inactiveStats.sumStatistic(StNumIndexWildSeeks, keyManager->queryWildSeeks()-startWildSeeks);
```

Comment:
these existing stats are now also going to be fetched from the context logger, so this old mechanism should be removed afaics.